buggy_script.py

```
@@ -0,0 +1,48 @@
+import time
+
+# 1. 逻辑/语法错误：函数定义缺少冒号，且缩进不规范
+def calculate_discount(price, discount)
+    final_price = price * (1 - discount)
+   return final_price  # 缩进错误
+
+# 2. 陷阱：使用可变对象（列表）作为默认参数
```

Ruff (0.14.10)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)